README.md

```diff
@@ -18,7 +18,7 @@ This repository uses `lerna` and `typescript`
 cd path-to/folder-name
 git clone git@github.com:telekom/scale.git
 cd scale
-git checkout development
+git checkout master
```

Comment:
Actualy we can remove this line I think, as `master` will be checked out by default 🙂

Reply:
ok. i will remove it.

Reply:
removed it